ajax/libs/swipejs/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "swipejs",
+  "version": "2.2.4",
+  "filename": "swipe.min.js",
+  "description": "Swipe is the most accurate touch slider",
+  "author": "Brad Birdsall",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lyfeyaj/swipe.git"
+  },
+  "keywords": [
+    "swipe",
+    "slider",
+    "touch"
+  ],
+  "homepage": "http://lyfeyaj.github.io/swipe/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/lyfeyaj/swipe.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "swipe*"
+        ]
+      },
+      {
+        "basePath": "build",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
```

Comment:
I think we also need `style.css` whose `basePath` field is `""`.
Ref: https://github.com/lyfeyaj/swipe#style
> Swipe requires the following styles to be added to your stylesheet: ...

It's the same as `style.css`.